pint_server/pint_db_migrate/versions/346dbf517c0e_in_the_beginning.py

```diff
@@ -0,0 +1,156 @@
+    sa.Column('deprecatedon', sa.Date),
+    sa.Column('deletedon', sa.Date),
+    sa.Column('changeinfo', sa.String(255)),
+
+    sa.Column('project', sa.String(50)),
+)
+
+microsoftimages = sa.Table(
+    'microsoftimages', meta,
+    sa.Column('name', sa.String(255), primary_key=True),
+    sa.Column('state', image_state),
+    sa.Column('replacementname', sa.String(255)),
+    sa.Column('publishedon', sa.Date, primary_key=True),
+    sa.Column('deprecatedon', sa.Date),
+    sa.Column('deletedon', sa.Date),
+    sa.Column('changeinfo', sa.String(255)),
+
+    sa.Column('environment', sa.String(50), primary_key=True),
+    sa.Column('urn', sa.String(100))
+)
+
+amazonservers = sa.Table(
+    'amazonservers', meta,
+    sa.Column('type', server_type),
+    sa.Column('shape', sa.String(10)),
+    sa.Column('name', sa.String(100)),
+    sa.Column('ip', postgresql.INET, primary_key=True),
+    sa.Column('region', sa.String(100), primary_key=True),
+)
+
+googleservers = sa.Table(
+    'googleservers', meta,
+    sa.Column('type', server_type),
+    sa.Column('shape', sa.String(10)),
+    sa.Column('name', sa.String(100)),
+    sa.Column('ip', postgresql.INET, primary_key=True),
+    sa.Column('region', sa.String(100), primary_key=True),
+)
+
+microsoftservers = sa.Table(
+    'microsoftservers', meta,
+    sa.Column('type', server_type),
+    sa.Column('shape', sa.String(10)),
+    sa.Column('name', sa.String(100)),
+    sa.Column('ip', postgresql.INET, primary_key=True),
+    sa.Column('region', sa.String(100), primary_key=True),
+)
+
+microsoftregionmap = sa.Table(
+    'microsoftregionmap', meta,
+    sa.Column('environment', sa.String(50), primary_key=True),
+    sa.Column('region', sa.String(100), primary_key=True),
+    sa.Column('canonicalname', sa.String(100), primary_key=True),
+)
+
+oracleimages = sa.Table(
+    'oracleimages', meta,
+    sa.Column('name', sa.String(255), primary_key=True),
+    sa.Column('state', image_state),
+    sa.Column('replacementname', sa.String(255)),
+    sa.Column('publishedon', sa.Date, primary_key=True),
+    sa.Column('deprecatedon', sa.Date),
+    sa.Column('deletedon', sa.Date),
+    sa.Column('changeinfo', sa.String(255)),
+    sa.Column('id', sa.String(100), primary_key=True),
+    sa.Column('replacementid', sa.String(100)),
+)
+
+versions = sa.Table(
+    'versions', meta,
+    sa.Column('tablename', sa.String(100), primary_key=True),
+    sa.Column('version', sa.Numeric, nullable=False),
+)
+
+
+def upgrade():
+    # create the enums and tables if they don't exist already
+    for obj in [server_type, image_state, alibabaimages, amazonservers,
+                amazonimages, googleimages, googleservers, microsoftimages,
+                microsoftregionmap, microsoftservers, oracleimages,
+                versions]:
+        obj.create(op.get_bind(), checkfirst=True)
+
+
+def downgrade():
+    for tablename in ['alibabaimages', 'amazonimages', 'amazonservers',
```

Comment:
can we use the table object and its `name` attribute ?

as `[amazonimages.name, alibabaimages.name,....]` 

It would be less prone to errors having to type the name once, at the creation of the table

Reply:
Good idea!